src/reducers/issues.reducer.js

```diff
@@ -35,7 +48,21 @@ export default handleActions(
     [SET_COMMENTS]: (state: IssuesState, { payload }) =>
       set('issueComments', payload.comments, state),
     [REMOVE_COMMENTS]: state =>
-      set('issueComments', initialState.issueComments, state)
+      set('issueComments', initialState.issueComments, state),
+    [SET_ISSUES_PAGING]: (state, { payload }) => {
+      const pageCount = get('header.last.page', payload);
+      return set('pageCount', +pageCount, state);
+    },
+    [SET_CURRENT_PAGE]: (state, { payload }) =>
+      set('currentPage', payload.currentPage, state),
+    [SET_SORT_STATE]: (state, { payload }) =>
+      flow([set('issuesState', payload.issuesState), set('currentPage', 1)])(
```

Comment:
``
```suggestion
      flow([
        set('issuesState', payload.issuesState), 
        set('currentPage', 1)]
      )(state)
```

more readable IMHO